ui/src/App.js

```diff
@@ -31,6 +31,7 @@ class App extends Component {
               });
           }
       }.bind(this);
+      // Map of facetName:[facetValues] pairs
       this.searchFilters = new Map();
```

Comment:
Can searchFilters be renamed to filterMap?

Reply:
Done